.eslintrc

```diff
@@ -18,6 +18,15 @@
     "@typescript-eslint/consistent-type-imports": "off",
     "@typescript-eslint/no-floating-promises": "off",
     "@typescript-eslint/no-unused-expressions": "off",
-    "mocha/no-mocha-arrows": "off"
+    "@typescript-eslint/return-await": "off",
+    "mocha/no-mocha-arrows": "off",
+    "@typescript-eslint/strict-boolean-expressions": [
+      2,
```

Comment:
How does the 2 fit in here?

Reply:
it's the same as "error" - replaced for consistency

Reply:
[1a7a0f0](https://github.com/ardriveapp/promise-cache/pull/2/commits/1a7a0f0b0dbb5cd3d4751ef72675c9de56f2dbd3)